template/package.json

```diff
@@ -96,6 +96,12 @@
   "nodemonConfig": {
     "ignore": ["*-css.js", "*.css"]
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm-run-all preCommit test linters bundler postinstall",
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
+    }
+  },
```

Comment:
This is the old husky method - need to update to use Husky 6 scripting.

Reply:
Branch just needed to be rebased and this was addressed.